### PR TITLE
Added habitat_baselines config namespace for the habitat-baselines

### PR DESCRIPTION
**Pull request by rpartsey**
## Motivation and Context

Making a habitat_baselines namespace for the habitat-baselines configuration keys.

## Types of changes

- Docs change / refactoring / dependency upgrade
- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [x] I have completed my CLA (see **CONTRIBUTING**)
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: examples/tutorials/colabs/Habitat_Lab.ipynb [hunk 1]
```diff
@@ -571,7 +571,7 @@
     "    config.habitat.simulator.type = \"Habitat-Sim-v0\"\n",
     "```\n",
     "\n",
-    "Paper: [https://arxiv.org/abs/1912.06321](https://arxiv.org/abs/1912.06321)"
+    "Paper: [https://arxiv.org/abs/1912.06321](https://arxiv.org/abs/1912.06321)\n"
```
Comment (vincentpierre) [inline examples/tutorials/colabs/Habitat_Lab.ipynb:574]:
```suggestion
    "Paper: [https://arxiv.org/abs/1912.06321](https://arxiv.org/abs/1912.06321)"
```

File: examples/tutorials/colabs/Habitat_Lab.ipynb [hunk 2]
```diff
@@ -595,4 +595,4 @@
  },
  "nbformat": 4,
  "nbformat_minor": 4
-}
+}
```
Comment (vincentpierre) [inline examples/tutorials/colabs/Habitat_Lab.ipynb:598]: Is that change needed? I think pre-commit should complain about this.

Comment (rpartsey) [reply examples/tutorials/colabs/Habitat_Lab.ipynb:598]: Oh, no
This is some how automatically done by my IDE. I'll fix.

File: habitat-baselines/habitat_baselines/config/rearrange/spap_pick.yaml [hunk 3]
```diff
@@ -1,37 +1,38 @@
...
-  mp_sim_type: "Priv"
-  video_dir: 'data/vids'
-  debug_dir: "data/mp_test"
-  count_obj_collisions: True
-  grasp_gen_is_verbose: True
-  ik_dist_thresh: 0.1
+  SENSE_PLAN_ACT:
+    verbose: True
+    RUN_FREQ: 2
+    KINEMATIC_CTrl: True
```
[the remark this answers is not in the document]
Comment (rpartsey) [reply habitat-baselines/habitat_baselines/config/rearrange/spap_pick.yaml:7]: I'll fix.

File: habitat-lab/habitat/config/default.py [hunk 4]
```diff
@@ -1025,6 +1025,7 @@ def get_config(
 
         for config_path in config_paths:
             config_path = _get_full_config_path(config_path)
+            print(config_path)
```
[the remark this answers is not in the document]
Comment (rpartsey) [reply habitat-lab/habitat/config/default.py:1028]: Good catch

File: habitat-baselines/habitat_baselines/config/rearrange/spap_pick.yaml [hunk 5]
```diff
@@ -1,37 +1,38 @@
...
-  TIMEOUT: 3
-  EXEC_EE_THRESH: 0.1
-  # "Priv" or "Reg"
-  mp_sim_type: "Priv"
-  video_dir: 'data/vids'
-  debug_dir: "data/mp_test"
-  count_obj_collisions: True
-  grasp_gen_is_verbose: True
-  ik_dist_thresh: 0.1
+  SENSE_PLAN_ACT:
```
[the remark this answers is not in the document]
Comment (rpartsey) [reply habitat-baselines/habitat_baselines/config/rearrange/spap_pick.yaml]: done

File: habitat-baselines/habitat_baselines/config/rearrange/spap_pick.yaml (hunk 3 above)
[the remark this answers is not in the document]
Comment (rpartsey) [reply habitat-baselines/habitat_baselines/config/rearrange/spap_pick.yaml]: done

File: habitat-baselines/habitat_baselines/config/rearrange/spap_pick.yaml [hunk 6]
```diff
@@ -1,37 +1,38 @@
...
+    verbose: True
+    RUN_FREQ: 2
+    KINEMATIC_CTrl: True
+    N_GRASPS: 100
+    MP_OBJ: True
+    MP_MARGIN: null
+    MP_RENDER: True
+    TIMEOUT: 3
+    EXEC_EE_THRESH: 0.1
```
[the remark this answers is not in the document]
Comment (rpartsey) [reply habitat-baselines/habitat_baselines/config/rearrange/spap_pick.yaml]: done

File: habitat-baselines/habitat_baselines/config/tp_srl_test/tp_srl_test.yaml [hunk 7]
```diff
@@ -1,90 +1,91 @@
...
+        name: "HierarchicalPolicy"
+        high_level_policy:
+          name: "GtHighLevelPolicy"
+        obs_transforms:
+          enabled_transforms: ['AddVirtualKeys']
+          ADD_VIRTUAL_KEYS:
+            "nav_to_skill": 8
+            "object_to_agent_gps_compass": 2
+        DEFINED_SKILLS:
+          NN_PICK:
```
[the remark this answers is not in the document]
Comment (rpartsey) [reply habitat-baselines/habitat_baselines/config/tp_srl_test/tp_srl_test.yaml]: done

File: habitat-lab/habitat/config/default.py (hunk 4 above)
[the remark this answers is not in the document]
Comment (rpartsey) [reply habitat-lab/habitat/config/default.py]: done

File: habitat-baselines/habitat_baselines/il/data/data.py [hunk 8]
```diff
@@ -51,9 +51,9 @@ def __init__(
             self.q_vocab = self.env._dataset.question_vocab
             self.ans_vocab = self.env._dataset.answer_vocab
 
-            self.eval_save_results = config.EVAL_SAVE_RESULTS
+            self.eval_save_results = config.habitat_baselines.eval_SAVE_RESULTS
```
[the remark this answers is not in the document]
Comment (rpartsey) [reply habitat-baselines/habitat_baselines/il/data/data.py]: done

File: examples/tutorials/colabs/Habitat_Lab.ipynb (hunk 2 above)
Comment (rpartsey) [reply examples/tutorials/colabs/Habitat_Lab.ipynb]: done

**Review by vincentpierre — APPROVED** [review]

**Comment by vincentpierre** [comment]
Looks like some of the tests failed, but it looks like the issue is due to the tests using old keys. Should be easy to fix.

**Comment by vincentpierre** [comment]
Probably some of the documentation and comments in habitat_baselines would need to use the new keys as well. Have a look to make sure they are alright.
